section/doc/tactic-developer/developer/watch-folder-service/index.txt

```diff
@@ -94,37 +111,49 @@ An alias and the absolute path for the device directory you added will need to b
 
-7)	Save and exit the changes to the file.
-8)	Go to the /home/tactic/tactic_data/config directory (or TACTIC install directory where tactic_data/config is located).
-9)	Type: vi tactic-conf.xml
+*NOTE*: The alias does not need to be the last folder in the directory. It can be a generic alias that would clearly define this directory as a unique path.
+
+7.  Save and exit the changes to the file.
+8.  Go to the /home/tactic/tactic_data/config directory (or TACTIC install directory where tactic_data/config is located).
+9.  Type: vi tactic-conf.xml
 This will use the Vim Editor to open the tactic-conf.xml file.
-10)	Under the <checkin> tag, there is a tag called <asset_base_dir> that needs to be set. To set the <asset_base_dir>, define an alias for the same full directory path you defined in the tactic.conf file:
+10.  Under the <checkin> tag, there is a tag called <asset_base_dir> that needs to be set. To set the <asset_base_dir>, define an alias for the same full directory path you defined in the tactic.conf file:
 
+----------------------------------------------
 <asset_base_dir>{
     "sample": "/example/directory/path/assets"
 }</asset_base_dir>
+----------------------------------------------
+
 
-11)	You will then need to set the <web_base_dir> under the <checkin> tag. The directory that you define for the <web_base_dir> is the directory as seen by the browser and is meant as TACTIC configuration to view the assets. The alias defined needs to be same alias defined in the <asset_base_dir>. The directory for the <web_base_dir> needs to be the same as the alias defined in the configuration for the web server (in the tactic.conf file):
+11.  You will then need to set the <web_base_dir> under the <checkin> tag. The directory that you define for the <web_base_dir> is the directory as seen by the browser and is meant as TACTIC configuration to view the assets. The alias defined needs to be same alias defined in the <asset_base_dir>. The directory for the <web_base_dir> needs to be the same as the alias defined in the configuration for the web server (in the tactic.conf file):
 
+------------------------
 <web_base_dir>{
     "sample": "/example"
 }</web_base_dir>
+------------------------
 
-12)	Save the changes and close the file.
-13)	Restart the web server and TACTIC to have the changes take effect. To do this, if using a shell, login as the “root” user, then type the following to restart the web server:
-	
-service httpd restart (CentOS)
+12.  Save the changes and close the file.
+13.  Restart the web server and TACTIC to have the changes take effect. To do this, if using a shell, login as the “root” user, then type the following to restart the web server:
++	
+        service httpd restart (CentOS)
++
 	OR
++
 	service apache2 restart (Debian/Ubuntu Linux)
-	
-Then, to restart TACTIC:
++	
+        Then, to restart TACTIC:
++
 	service tactic restart
 
```

Comment:
For this Section 1.6, please make sure the numbering of each step is coming out correctly in the documents. It seems as if they are numbered correctly here, but not showing properly when the books are generated.